mobly/controllers/android_device_lib/sl4a_client.py

```diff
@@ -29,6 +29,10 @@
 
 class Sl4aClient(jsonrpc_client_base.JsonRpcClientBase):
 
+    def __init__(self, adb_proxy):
+      super(Sl4aClient, self).__init__(adb_proxy)
+      self.app_name = 'sl4a'
```

Comment:
How about upper case 'SL4A' here?
Since this token often appears at the beginning of a log message.